webapp/components/interop-dashboard.js

```diff
@@ -672,5 +708,74 @@ class InteropDashboard extends PolymerElement {
     this.$.toggleStable.setAttribute('aria-pressed', true);
     this.$.toggleExperimental.setAttribute('aria-pressed', false);
   }
+
+  getSortIcon(index) {
+    if (this.toggleStates[index]) {
+      return '/static/expand_more.svg';
+    }
+    return '/static/expand_less.svg';
+  }
+
+  sortRows(rows, index, tableState) {
+    if(index === 0) {
+      const sortedFeatureOrder = [];
+
+      // Alphabetize by focus area column
+      if(tableState.sortColumn === -1) {
+        const rowNames = [];
+        for(let i = 0; i < rows.length; i++) {
+          const feature = rows[i];
+          rowNames[i] = [feature, this.getRowInfo(feature, 'description')];
+        }
+        rowNames.sort((a, b) => {
+          a[1] - b[1];
+        });
+
+        for (let i = 0; i < rowNames.length; i++) {
+          sortedFeatureOrder[i] = rowNames[i][0];
+        }
+        // List in ascending order by score
+      } else if (tableState.sortColumn >= 0) {
+        const individualScores = [];
+
+        for (let i = 0; i < rows.length; i++) {
+          const feature = rows[i];
+          individualScores[i] = [feature, parseFloat(this.getBrowserScoreForFeature(tableState.sortColumn, feature))];
```

Comment:
This is _technically_ doing what you want, but `getBrowserScoreForFeature()` is returning a string-formatted version of the browser score and you are having to parse that formatted string back into a number. Ideally, you should take the way that `getBrowserScoreForFeature()` is obtaining the number and implement it separately without needing to handle all of this typecasting.
Another possibility is creating a `formatBrowserScore()` method that takes the browser number score and formats it like a string. Then, you could refactor `getBrowserScoreForFeature()` to return the raw browser score number. (Hopefully I explained that well)